### ISSUE TITLE
Could not resolve host: api-content.dropbox.com

### ISSUE THREAD

**Issue by skuppuraj** [BenExile/Dropbox#99]
I am frequently getting this error while uploading file to drop box.This issue is occur only in shared server, dedicated server working fine. I contacted my host they said all are fine. They are reply 
We still able to telnet to api-content.dropbox.com on port 80 and 443. You can verify the same with logs given below.
# 

telnet api-content.dropbox.com 80
Trying 45.58.70.36...
Connected to api-content.dropbox.com.

telnet api-content.dropbox.com 443
Trying 45.58.70.36...
Connected to api-content.dropbox.com

My server not support IPV6

I tried all solution from google also I used new content URl (content.dropboxapi.com) same error.I don't how to resolve this . What can I change to my host?.
So can any one please assist to work

**Comment by ghost** [comment]
Where is that message that says "the server not support ipv6" coming from?

What happens when you `nslookup content.dropboxapi.com` from your virtual server?

Can you add an alias for the api endpoint on your virtual server?

**Comment by skuppuraj** [comment]
In my server not support IPV6 because it shared hosting .

When I run **nslookup content.dropboxapi.com** I got this output 

nslookup content.dropboxapi.com
Server: 75.98.161.224
Address:    75.98.161.224#53

Non-authoritative answer:
content.dropboxapi.com  canonical name = block-api.dropbox-dns.com.
block-api.dropbox-dns.com   canonical name = block-api-iad.dropbox-dns.com.
Name:   block-api-iad.dropbox-dns.com
Address: 45.58.70.36

**Comment by ghost** [comment]
Yeah, but ... that's OK, right? Dropbox isn't IPv6. your server isn't IPv6. so they should have no problem.

I don't think it's the IPv-whatever part which is causing the problem.
